inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/MD5Utils.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.util;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.shiro.crypto.hash.SimpleHash;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * MD5 encryption and decryption utils.
+ */
+@Slf4j
+@UtilityClass
+public class MD5Utils {
+
+    private static final char[] hexDigits = {'0', '1', '2', '3', '4', '5',
+            '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+
+    /**
+     * Get MD5 from the given string.
+     *
+     * @param source string to be encrypted
+     * @return MD5 string after encrypt
+     */
+    public static String getMD5String(String source) {
+        if (source == null) {
+            return null;
+        }
+
+        String retString = null;
+        try {
+            MessageDigest md = MessageDigest.getInstance("MD5");
```

Comment:
MD5 may cause the code scan to emit a security warning like:
https://github.com/apache/inlong/pull/4372/files#annotation_3899842325

Suggest to use SHA.

Reply:
This is old behavior. Subsequent plans to fix this issue.